cluster/gce/config-default.sh

```diff
@@ -407,6 +407,9 @@ SCHEDULING_ALGORITHM_PROVIDER="${SCHEDULING_ALGORITHM_PROVIDER:-}"
 # Optional: install a default StorageClass
 ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 
+# Optional: install volume snapshot CRDs
+ENABLE_VOLUME_SNAPSHOTS="${ENABLE_VOLUME_SNAPSHOTS:-true}"
```

Comment:
I'm not familiar with how "cluster/addons" is used. Looking at this change makes it sound like those addons explicitly have to be enabled?

Can that be done when using `hack/local-up-cluster.sh`? I ran it from your branch for this PR, but the resulting cluster didn't have the snapshot CRDs.

Reply:
That needs a separate PR.  I'll work on that.